### PR TITLE
Revise cluster-pause/start tests

### PR DESCRIPTION
**Pull request by benjaminjb**
**Checklist:**
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change, as applicable?
 - [ ] Have you tested your changes on all related environments with successful results, as applicable?
   - [ ] Have you added automated tests?

**Type of Changes:**
 - [ ] New feature
 - [ ] Bug fix
 - [ ] Documentation
 - [x] Testing enhancement
 - [ ] Other

**What is the current behavior (link to any open issues here)?**

**What is the new behavior (if this is a feature change)?**
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

* Use files for legibility
* Add describe/log collectors to every assert

**Other Information**:

### CODE REVIEW

**Comment by benjaminjb** [comment]
This is higher priority.

File: testing/kuttl/e2e/cluster-pause/files/01-pause-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: cluster-pause
+spec:
+  paused: true
+  instances:
+    - name: instance1
+      # We set replicas to 2, but this won't result in a new replica until we resume
+      replicas: 2
```
Comment (jmckulk) [inline testing/kuttl/e2e/cluster-pause/files/01-pause-cluster.yaml:10]: Is there another change that we could make that wouldn't require creating a new pod? :thinking:

**Review by jmckulk — APPROVED** [review]
lgtm

File: testing/kuttl/e2e/cluster-pause/files/01-pause-cluster.yaml (hunk 1 above)
Comment (benjaminjb) [reply testing/kuttl/e2e/cluster-pause/files/01-pause-cluster.yaml]: I changed it from a replica bump to changing the type of service. Better?

**Review by jmckulk — APPROVED** [review]
I like the service change :+1: hopefully it speeds up tests

**Pull request merged**